addons/account/models/account_move.py

```diff
@@ -621,6 +625,25 @@ def _compute_is_move_sent(self):
         for move in self:
             move.is_move_sent = bool(move.invoice_pdf_report_id)
 
+    def _compute_is_being_sent(self):
+        self.is_being_sent = False
+        if self.ids:
+            self.env['account.move.send'].flush_model(['move_ids', 'mode'])
+            self.env.cr.execute(
+                """
+                SELECT move_send_rel.account_move_id
+                  FROM account_move_send send
+                  JOIN account_move_account_move_send_rel move_send_rel
+                    ON move_send_rel.account_move_id in %(move_ids)s
+                   AND send.id = move_send_rel.account_move_send_id
+                 WHERE send.mode != 'done'
+                """,
+                params={'move_ids': tuple(self.ids)}
+            )
+            test = self.env.cr.fetchall()
+            for move_id in test:
+                self.browse(move_id).is_being_sent = True
```

Comment:
It works by chance because `browse` accepts a list of ids too but here, it's missleading:
`test' is in fact a list of list
'move_id' is in fact a singleton list with an id